app/views/pages/index.html.erb

```diff
@@ -1,9 +1,30 @@
-<div class="btn-group pull-right" data-toggle="buttons">
-    <% Filter.list.each do |filter| %>
-        <label id="<%= filter.filter_id %>" class="btn btn-primary" onclick="localStorage.setItem('jobfilter', '<%= filter.filter_id %>'); window.location = '?jobfilter=<%= filter.filter_id %>';" >
-          <input type="radio" name="<%= filter.filter_id %>" id="option_user" > <%= filter.title %>
-        </label>
-    <% end %>
+<div class="row pull-right">
+  <div class="div-inline dropdown">
+    <button class="btn btn-primary dropdown-toggle" type="button" id="dropdown-menu-filter" data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
+      <span id="selected-filter-label"></span>
+      <span class="caret"></span>
+    </button>
+    <ul class="dropdown-menu" aria-labelledby="dropdown-menu-filter">
+      <% Filter.list.each do |filter| %>
+          <%= '<li class="divider"></li>'.html_safe if filter == Filter.list.last %>
+          <li class="filter-dropdown-option" id="filter-id-<%= filter.filter_id %>" onclick="set_filter_id('<%= filter.filter_id %>');" data-title="<%= filter.title %>"><a href="#"><%= filter.title %></a></li>
```

Comment:
Also, I know this may throw a wrench in the saving state portion of the code, but I don't necessarily agree with that since we are using query parameters (another form of saving state). It would reduce your code if we didn't save all this to localStorage and people can still bookmark the links with the query params.